article/helpers.py

```diff
@@ -133,40 +146,40 @@ def retrieve_article_uuids(self):
         uuids = self.request.session.get(self.SESSION_KEY, [])
         return frozenset(uuids)
 
+    def clear(self):
+        self.session[self.SESSION_KEY] = []
+
 
 class DatabaseArticlesReadManager(BaseArticleReadManager):
 
-    def __init__(self, request):
-        super().__init__(request)
-        self.transfer_article_progress()
+    article_uuids = frozenset()
 
     def persist_article(self, article_uuid):
         response = api_client.exportreadiness.create_article_read(
             article_uuid=article_uuid,
             sso_session_id=self.request.sso_user.session_id,
         )
-        assert response.ok, response.content
+        log_response(response)
```

Comment:
API will return 400 if the article has already been seen, but that's fine given this facilitates:
 - not having to retrieve a list of already viewed articles
 - not creating duplicate article views